lib/phoenix_live_reload/frame.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Phoenix.LiveReloader.Frame do
+  phoenix_path = Application.app_dir(:phoenix, "priv/static/phoenix.js")
+  reload_path = Application.app_dir(:phoenix_live_reload, "priv/static/phoenix_live_reload.js")
+  @external_resource phoenix_path
+  @external_resource reload_path
+
+  @phoenix_js File.read!(phoenix_path) |> String.replace("//# sourceMappingURL=", "// ")
```

Comment:
Every time you read this module attribute, you are injecting it into the reader, which means we are probably making this module a couple kB larger than it needs to be. Make `phoenix_js()` a function instead and reuse it from everywhere.

Reply:
In fact, I don't think any of those calls need to be macros and we can likely keep the code inlined in the previous module to avoid confusion.

Reply:
Gotcha! I was following the module attribute pattern from the previous version which is why I switched to macros. Are you saying it's fine to just have all these be functions?

Reply:
Yes, before we were using the attributes because it was in the same module. But if it is in the same module, it should be a single function so we don't embed a separate copy of phoenix.js every time the module attribute or the macro is referenced. :)